Solutions/Windows2/TinyCLR/Various.cpp

```diff
@@ -437,6 +437,8 @@ void HAL_Windows_FastSleep( INT64 ticks )
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 
+#pragma unmanaged

```

Comment:
prefer using #pragma managed(push, off) + #pragma managed(pop) instead of directly forcing it to managed. This allows the code to work correctly even if built in a purely unmanaged configuration.